packages/bierzo-wallet/src/logic/csvBuilder/index.ts

```diff
@@ -0,0 +1,52 @@
+export interface CsvRow {
+  readonly id: string;
+  readonly recepient: string;
+  readonly sender: string;
+  readonly quantity: string;
+  readonly fractionalDigits: string;
+  readonly tokenTicker: string;
+  readonly feeQuantity: string;
+  readonly feeFractionalDigits: string;
+  readonly feeTokenTicker: string;
+  readonly time: string;
+  readonly note: string;
+  [key: string]: string;
+}
+
+class CsvRepresentation {
+  private readonly _fields: readonly string[];
```

Comment:
Let's drop this field

Reply:
I want to preserve order of the header fields.And use it for row values.

Reply:
And we can manipulate order of the fields using this variable